galsim/gsobject.py

```diff
@@ -1610,7 +1610,7 @@ def drawFFT_makeKImage(self, image, wmult=1.):
 
         if Nk > self.gsparams.maximum_fft_size:
             raise RuntimeError(
-                "drawFFT requires an FFT that is too large: %s."%Nk +
+                "drawFFT requires an FFT that is too large: %s. "%Nk +
```

Comment:
I have a tendency to use `%s` for everything, since it always works correctly regardless of what type the variable is.  Unless I want to do something like `%04d` or similar.  But `%d` would be equivalent here.